tests/test_other.py

```diff
@@ -10484,3 +10484,28 @@ def test_metadce_wasm2js_i64(self):
 }''')
     run_process([EMCC, 'src.cpp', '-O3', '-s', 'WASM=0'])
     self.run_js('a.out.js')
+
+  def test_DETERMINISTIC(self):
```

Comment:
So we have other tests that use uppercase names like this?   I have slight preference for keeping the names lowercase unless there is president.

Reply:
Hmm, I prefer the uppercase when it matches something like a settings value, so that it's easy to search for. But it is annoying that it's ambiguous, and better to be consistent. So let's do lowercase as you suggest.